### ISSUE TITLE
NoMethodError: undefined method `property' for Chef::Resource::WindowsFeatureDism:Class 

### ISSUE THREAD

**Issue by Benjamin-Connelly** [chocolatey/chocolatey-cookbook#132]
### Cookbook version
Chocolatey:
0.6.2 to current. 
I have tried many versions

Windows:
2.1.1 to current
I have tried many versions

### Chef-client version
Opsworks 12.2

### Platform Details
AWS Opsworks Server 2012

### Scenario:
Current recipe with packages I would like to install:

```#Install Chocolatey
include_recipe 'chocolatey'

#Install MS Build Tools
chocolatey 'microsoft-build-tools' do
  version '14.0.23107.10'
  options "--ignore-checksums"
  action :install
end

#Install MS Build Tools
chocolatey 'dotnet4.6-targetpack' do
  options "--ignore-checksums"
  action :install
end

#Install nugget cli
chocolatey 'nuget.commandline' do
  options "--ignore-checksums"
  action :install
end

#Install some packages
%w{sysinternals 7zip notepadplusplus googlechrome console2 }.each do |pack|
  chocolatey pack
end
```

### Steps to Reproduce:
This is a vanilla setup that fails on each run. 

### Expected Result:
Install packages

### Actual Result:
Stacktrace.out and Opsworks log - [Gist](https://gist.github.com/Benjamin-Connelly/9a7e1b5524abaaa6c2b6df2d7d8f6f65).

**Comment by ferventcoder** [comment]
It's more helpful for managing packages to use the Chef built in `chocolatey_package` and to use this cookbook for managing install and configuration of Chocolatey itself.

**Comment by ferventcoder** [comment]
Seems like a bug, I don't want to leave you hanging on this, but I know we have deprecated the cookbook chocolatey methods.

**Comment by Benjamin-Connelly** [comment]
Thank you for your reply @ferventcoder . Opsworks is at only at `chef_version 12.2` in Opsworks. The `chocolatey_package` call is not implemented until  `chef_version 12.7`. 

I am realizing that his is wide spread with any cookbooks for Windows in the supermarket. I was looking forward to using `chocolatey` and `iis` community cookbooks and not hacking this together myself with powershell commands.

**Comment by tas50** [comment]
This cookbook no longer ships the package resource since that was included within Chef 12.7 which we released Feb 2016. If you're not yet on Chef 12.7 or later you'll want to upgrade, but you shouldn't see this issue anymore, especially since the 2.0 release of the cookbook removed the package resource entirely.

**Comment by Benjamin-Connelly** [comment]
Unfortunately Opsworks doesn't keep up with Chef versions.


Benjamin Connelly
Mobile: 415-917-0763
Email: benjamin.t.connelly@gmail.com
Resume: https://benjamin-connelly.github.io/
LInkedin: https://www.linkedin.com/in/benjaminconnelly
